doctor/docs/base.py

```diff
@@ -149,34 +240,40 @@ def get_json_object_lines(annotation: ResourceAnnotation,
         if request and prop in url_params:
             field = 'param'
 
-        types = [str(annotated_type.native_type.__name__)]
+        types = get_json_types(annotated_type)
         description = annotated_type.description
         obj_ref = ''
         if issubclass(annotated_type, Object):
-            resource_name = annotated_type.title
-            if resource_name is None:
-                class_name = annotated_type.__name__
-                resource_name = class_name_to_resource_name(class_name)
-            ALL_RESOURCES[resource_name] = annotated_type
-            obj_ref = ' See :ref:`resource-{}`.'.format(
-                '-'.join(resource_name.split(' ')).lower().strip())
-        elif (hasattr(annotated_type, 'items') and
```

Comment:
I had to change this to `issubclass(annotated_type, Array)` as Object types subclass dict which has an `items` attr.